homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -127,11 +130,6 @@ def available(self):
         """Return true when state is known."""
         return self._available
 
-    @property
-    def unique_id(self):
-        """Return the unique ID."""
-        return self._unique_id
-
     @property
     def air_quality_index(self):
         """Return the Air Quality Index (AQI)."""
```

Comment:
Please remove `temperature` and `humidity` properties below too.

Reply:
see comment down below